ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py

```diff
@@ -69,14 +69,14 @@
     - present
     type: str
     description:
-    - Choice for fibre channel operation.
+    - The fibre channel operation.
   wwnn_base_str:
     required: False
     type: str
     description:
-    - Base World Wide Node Name(WWNN). Six bytes expressed in hex separated by colons. Example:- 'af:32:f1'.
+    - Base WWNN. Six bytes expressed in hex separated by colons. Example:- 'af:32:f1'.
```

Comment:
Is it possible to write  as Example: "af:32:f1" instead?

Reply:
not possible.. ':' gives rendering error for ansible-doc command